### ISSUE TITLE
Feature request: calendar subscription

### ISSUE THREAD

**Issue by wohenbushuang** [meeting-room-booking-system/mrbs-code#3577]
**Is your feature request related to a problem? Please describe.**
I hope that users can add the meetings to their personal calendar app such as calendar app on their Android or iOS mobile/pad, Outlook, web mail system, etc.

**Describe the solution you'd like**
An online calendar synchronization method for subscription, instead of .ics attachment. Because users are not familiar with settings to automatically deal with .ics attachment in their various mail app, and it's impossible for me to study and teach them either.

**Describe alternatives you've considered**
I found some related subscription solutions:
* webcal, which will give a url with .ics surfix. dynamically generate the latest when visit.
* caldav service, which is based on webdav, which is based on HTTP
* Exchange or Exchange ActiveSync service, though might be too heavy

**Comment by campbell-m** [comment]
Presumably users woud just want to add their own meetings?  Would it also be desirable to be able to subscribe to the calendar for a particular room or area?

**Comment by wohenbushuang** [comment]
Right, users want to subscribe their own meetings.

Further, maybe someone else need rooms or areas can also have their own subscription calendar for janitors, or be used as data source by infomation screen devices.

**Comment by embeh** [comment]
For us it would be very useful for users to be able to subscribe to specific rooms so they can import the room bookings into their own calendar with updates as they come.

**Comment by nicktech** [comment]
Same here. My users have wanted to subscribe to their rooms for a long time. Up until now, the fact that google would automatically add attached .ics files from the notifications to their calendar was a workaround. But that has stopped working as well. I think google has tightened up that feature and doesn't like that the .ics file contains METHOD:REQUEST.

**Comment by campbell-m** [comment]
> But that has stopped working as well. I think google has tightened up that feature and doesn't like that the .ics file contains METHOD:REQUEST.

I agree that calendar subscription would be a useful feature, but I think it should still be possible to have Google automatically add events to the calendar.  You just need to make sure it has been enabled - see [this article](https://support.google.com/calendar/answer/6084018?hl=en&co=GENIE.Platform%3DDesktop).
